certbot-apache/certbot_apache/tests/configurator_test.py

```diff
@@ -59,13 +59,14 @@ def test_prepare_no_install(self, mock_surgery, _init_augeas):
             self.assertRaises(errors.NoInstallationError, self.config.prepare)
             self.assertEqual(mock_surgery.call_count, 1)
 
-    @mock.patch("certbot_apache.augeas_configurator.AugeasConfigurator.init_augeas")
+    @mock.patch("certbot_apache.parser.ApacheParser.init_augeas")
```

Comment:
If we make the change I suggested of moving the handling of this exception to `init_augeas`, I think we should move this test to the parser tests.